### ISSUE TITLE
Better performance on test set than training set?

### ISSUE THREAD

**Issue by nizqleo** [erikwijmans/Pointnet2_PyTorch#92]
Hi! 
Thanks for your code!
But during testing, I found the accuracy of the model on the test set(the one labeled '-val', I suppose?) is always better than it on the training set. Is this normal? Or am I making mistakes with the code or understanding the situation wrong?

Thanks in advance!

**Comment by erikwijmans** [comment]
That will happen at the start of training due to regularization and data augmentation, but shouldn't persist once the model converges.
